tools/CompileReleaseNotes.py

```diff
@@ -243,10 +247,16 @@ def format_list_of_prs(prs):
     # Try to query GitHub for further information on the merged PRs
     if not args.no_github:
         import github
-        import tqdm
         gh = github.Github(args.github_token)
         gh_repo = gh.get_repo(args.github_repository)
-        for pr in tqdm.tqdm(merged_prs, desc="Downloading PR data", unit="PR"):
+        if args.silent:
+            prs_iterator = iter(merged_prs)
+        else:
+            import tqdm
```

Comment:
If I remember correctly there is some documentation that says this script requires `tqdm`. Since this appears to only be true if you aren't in `silent` mode, maybe update that when you squash?

Reply:
That's in the dev guide I add in https://github.com/sxs-collaboration/spectre/pulls :) Will update it there.

Reply:
:+1: